packages/kbn-management/cards_navigation/src/consts.tsx

```diff
@@ -135,10 +127,19 @@ export const appDefinitions: Record<AppId, AppDefinition> = {
     icon: 'gear',
   },
 
+  // Access section
+  [AppIds.API_KEYS]: {
```

Comment:
Previously, only search offered a nav link in the sidebar to API keys. Do we want to expose the API keys nav card in all 3 serverless project types?